.github_changelog_generator

```diff
@@ -1,5 +1,6 @@
 breaking_labels=breaking change
 add-sections={"docs":{"prefix":"**Documentation updates:**","labels":["documentation"]}}
+add-sections={"maintenance":{"prefix":"**Project maintenance**","labels":["maintenance"]}}
```

Comment:
```suggestion
add-sections={"maintenance":{"prefix":"**Project maintenance:**","labels":["maintenance"]}}
```
I hope this doesn't override the previous add-sections?

Reply:
It does! Thanks for the catch! Will fix